pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
```

Comment:
A few other suggestions for this file:

1. Let's use proto3, unless there's a specific reason not to?
2. I'd suggest a comment at the top explaining the purpose of these protos, along with the boilerplate header

Reply:
How can I set keys field to optional?

Reply:
Just don't set it to required.